--- a/start.php
+++ b/start.php
@@ -90,21 +90,22 @@
 	 * @author Leo Fajardo (@leorw)
 	 * @since 2.2.3
 	 */
-	$themes_directory      = fs_normalize_path( get_theme_root( get_stylesheet() ) );
-	$themes_directory_name = basename( $themes_directory );
-
-    // This change ensures that the condition works even if the SDK is located in a subdirectory (e.g., vendor)
-    $theme_candidate_sdk_basename = str_replace( $themes_directory . '/' . get_stylesheet() . '/', '', $fs_root_path );
-
-    // Check if the current file is part of the active theme.
-    $is_current_sdk_from_active_theme = $file_path == $themes_directory . '/' . get_stylesheet() . '/' . $theme_candidate_sdk_basename . '/' . basename( $file_path );
-    $is_current_sdk_from_parent_theme = false;
-
-    // Check if the current file is part of the parent theme.
-    if ( ! $is_current_sdk_from_active_theme ) {
-        $theme_candidate_sdk_basename     = str_replace( $themes_directory . '/' . get_template() . '/',
-            '',
-            $fs_root_path );
-        $is_current_sdk_from_parent_theme = $file_path == $themes_directory . '/' . get_template() . '/' . $theme_candidate_sdk_basename . '/' . basename( $file_path );
-    }
+	if ( ! function_exists( 'get_theme_root' ) ) {
+		$is_current_sdk_from_active_theme = false;
+		$is_current_sdk_from_parent_theme = false;
+	} else {
+		$themes_directory      = fs_normalize_path( get_theme_root( get_stylesheet() ) );
+		$themes_directory_name = basename( $themes_directory );
+		// This change ensures that the condition works even if the SDK is located in a subdirectory (e.g., vendor)
+		$theme_candidate_sdk_basename = str_replace( $themes_directory . '/' . get_stylesheet() . '/', '', $fs_root_path );
+		// Check if the current file is part of the active theme.
+		$is_current_sdk_from_active_theme = $file_path == $themes_directory . '/' . get_stylesheet() . '/' . $theme_candidate_sdk_basename . '/' . basename( $file_path );
+		$is_current_sdk_from_parent_theme = false;
+		// Check if the current file is part of the parent theme.
+		if ( ! $is_current_sdk_from_active_theme ) {
+			$theme_candidate_sdk_basename     = str_replace( $themes_directory . '/' . get_template() . '/', '', $fs_root_path );
+			$is_current_sdk_from_parent_theme = $file_path == $themes_directory . '/' . get_template() . '/' . $theme_candidate_sdk_basename . '/' . basename( $file_path );
+		}
+	}
 
 	$theme_name = null;
 	if ( $is_current_sdk_from_active_theme ) {