--- a/inc/git-updater/class-provider.php
+++ b/inc/git-updater/class-provider.php
@@ -96,7 +96,7 @@
 		$data->slug = $package->slug;
 		$data->filename = $package->file;
 		$data->description = substr( strip_tags( trim( $package->sections['description'] ) ), 0, 139 ) . '…';
-		$data->keywords = $package->readme_tags ? : array_values( $package->readme_tags ) : [];
+		$data->keywords = $package->readme_tags ? array_values( $package->readme_tags ) : [];
 		$data->sections = $package->sections;
 
 		// Parse link back out of author string.